pkg/metrics/otel.go

```diff
@@ -138,6 +151,14 @@ processors:
           # included by the 'regex include' filter above and is not included in
           # our Monarch metric definitions
           - kcc_resource_count_total
+  attributes/kubernetes:
+    actions:
```

Comment:
Add a comment like "Remove custom configsync metric labels. These labels are not registered with Monarch."

Reply:
Done